pages/_app.js

```diff
@@ -2,10 +2,24 @@ import "../styles/main.css";
 import SearchState from "../context/search/SearchState";
 import CartState from "../context/cart/CartState";
 import Navbar from "../components/Navbar";
+import {useRouter} from "next/router";
+import head from "next/Head"
 
 export default function ({ Component, pageProps }) {
+  const router = useRouter();
+
+  let canonicalUrl
+  if (process.env.production) {
```

Comment:
brug process.env.NODE_ENV i stedet for
hvis NODE_ENV er === "production" så brug production path, ellers brug localhost path

Reply:
Det behøver ikke være NODE_ENV bare det er en environment variable der bliver sat, således at man kan opsætte conditions baseret på det environment vi kører applikationen i. Fx "test", "production", "development"